pkg/operator/apiserver/controller/auditpolicy/auditpolicy_controller.go

```diff
@@ -48,13 +47,13 @@ func NewAuditPolicyController(
 	c := &auditPolicyController{
 		controllerInstanceName: factory.ControllerInstanceName(name, "AuditPolicy"),
 		operatorClient:         operatorClient,
-		apiserverConfigLister:  apiserverConfigLister,
+		apiserverConfigLister:  configInformers.Config().V1().APIServers().Lister(),
 		kubeClient:             kubeClient,
 		targetNamespace:        targetNamespace,
 		targetConfigMapName:    targetConfigMapName,
 	}
 
-	return factory.New().WithSync(c.sync).WithControllerInstanceName(c.controllerInstanceName).ResyncEvery(10*time.Second).WithInformers(
+	return factory.New().WithSync(c.sync).WithControllerInstanceName(c.controllerInstanceName).ResyncEvery(1*time.Minute).WithInformers(
```

Comment:
It looks like the sync will be triggered by the informers anyway. At some point, we might remove the artificial resync altogether